bitnami/thanos/README.md

```diff
@@ -1340,16 +1340,16 @@ ruler:
         rules:
           - alert: "PrometheusDown"
             expr: absent(up{prometheus="monitoring/kube-prometheus"})
-metrics:
-  enabled: true
-  serviceMonitor:
-    enabled: true
```

Comment:
Could we get the metrics enabled or there is any issues with this ?

Reply:
ok will do there is not problem at all i remove them for debuging.